src/common/thumbnail-manager.js

```diff
@@ -21,7 +21,7 @@ class ThumbnailManager {
   }
 
   getThumbnail(time: number): ?ThumbnailInfo {
-    if (this._thumbnailConfig) {
+    if (this._isValidThumbnailConfig()) {
```

Comment:
@dan-ziv didn't you create an ImageTrack for Kaltura thumbnail as well?

Reply:
Currently no. I've thoughts about it, but I don't no if it's right - tracks are something that defined and comes from the manifest and this is not the case here. So it will maybe looks weird and buggy to see a track comes out of no where.
BTW can you think how to add track from outside of playkit? probably this is something that if we will do should be pass through the config...

Reply:
We already have something like this, we have external tracks which comes from config
BTW don't we get a list for thumbnail images?

Reply:
Yes, but in external tracks the config explicitly defines it - not like here when its happens "magically" since this data comes from the backend (usually, except some OTT partners).
Regarding your Q, I think you are confusing with posters retrieved from OTT provider, but in this case this is not URLs of Kaltura's thumbnail service so the `thumbsSprite` result will be an empty string (this is why the regex check).